Tools/TestWebKitAPI/Tests/WebKitCocoa/WritingTools.mm

```diff
@@ -224,9 +209,9 @@ - (void)focusDocumentBodyAndSelectAll
 - (PlatformWritingToolsResultOptions)allowedWritingToolsResultOptionsForTesting
 {
 #if PLATFORM(IOS_FAMILY)
-    return [(id<UITextInput_Staging130540028>)[self textInputContentView] allowedWritingToolsResultOptions];
+    return [(id<UITextInput>)[self textInputContentView] allowedWritingToolsResultOptions];
```

Comment:
I suspect the cast here isn't necessary, given the return value of `-textInputContentView`.

```suggestion
    return [[self textInputContentView] allowedWritingToolsResultOptions];
```